lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -797,6 +798,7 @@ class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
     auto resultTy = cast<ValueTensorType>(op.getType());
 
     Value inputZp, weightZp;
+    bool inputUnsigned, weightUnsigned = false;
```

Comment:
Are you expect to initialize both of these to `false`? I usually recommend to do the following for clarity
```
bool inputUnsigned = false;
bool weightUnsigned = false;
```